helm/alfresco-content-services/Chart.yaml

```diff
@@ -14,6 +14,8 @@ home: https://www.alfresco.com
 sources:
   - https://github.com/Alfresco/acs-deployment
 dependencies:
+  - name: alfresco-common
```

Comment:
this is better named as alfresco-content-common if this subchart is specific to content